README.md

```diff
@@ -1,2 +1,107 @@
 # go-cruise-control
-It's client library written in Golang for interacting with Linkedin Cruise Control using its HTTP API.
+
+It's client library (written in Golang) for interacting with
+[Linkedin Cruise Control](https://github.com/linkedin/cruise-control) using its HTTP API.
+
+## How to use it
```

Comment:
Though CC is quite fast at creating new version (so maintainers can update the latest CC version every day), but I think it would be beneficial to include the latest tested or verified CC versions in the readme.

Reply:
Agreed. I planned to do that, but forgot about it. Will do.

Reply:
Fixed.